opinions/spring_boot_opinions.go

```diff
@@ -285,3 +324,35 @@ func SpringApplicationProperties(ctx context.Context) map[string]string {
 	}
 	return nil
 }
+
+type SpringBootServiceIntent struct {
+	Id           string
+	LabelName    string
+	Dependencies sets.String
+}
+
+func (o *SpringBootServiceIntent) GetId() string {
+	return o.Id
+}
+
+func (o *SpringBootServiceIntent) Applicable(applied AppliedOpinions, metadata cnb.BuildMetadata) bool {
+	bootMetadata := NewSpringBootBOMMetadata(metadata)
+	for _, d := range bootMetadata.Dependencies {
+		if o.Dependencies.Has(d.Name) {
+			return true
+		}
+	}
+	return false
+}
+
+func (o *SpringBootServiceIntent) Apply(ctx context.Context, podSpec *corev1.PodTemplateSpec, metadata cnb.BuildMetadata) error {
+	bootMetadata := NewSpringBootBOMMetadata(metadata)
+	for _, d := range bootMetadata.Dependencies {
+		if o.Dependencies.Has(d.Name) {
+			podSpec.Labels[o.LabelName] = d.Name
```

Comment:
When we add support for multiple contains we should add something like a complementary annotation so we know which container wants the service.

Reply:
opened #31 to track